docs/polaris/data-science-workflows/python.md

```diff
@@ -1,88 +1,111 @@
 # Python
 
-## Conda
-We provide prebuilt `conda` environments containing GPU-supported builds of `torch`, `tensorflow` (both with `horovod` support for multi-node calculations), `jax`, and many other commonly-used Python modules. 
+We provide prebuilt `conda` environments containing GPU-supported builds of
+`torch`, `tensorflow` (both with `horovod` support for multi-node
+calculations), `jax`, and many other commonly-used Python modules.
 
-Users can activate this environment by first loading the `conda` module, and then activating the base environment.
+Users can activate this environment by first loading the `conda` module, and
+then activating the base environment.
 
 Explicitly (either from an interactive job, or inside a job script):
 
 ```bash
-$ module load conda
-$ conda activate base
-(base) $ which python3
-/soft/datascience/conda/2022-09-08/mconda3/bin/python3
+module load conda ; conda activate base
 ```
-In one line, `module load conda; conda activate`. This can be performed on a compute node, as well as a login node. 
 
-As of writing, the latest `conda` module on Polaris is built on Miniconda3 version 4.14.0 and contains Python 3.8.13. Future modules may contain entirely different major versions of Python, PyTorch, TensorFlow, etc.; however, the existing modules will be maintained as-is as long as feasible. 
+This will load and activate the base environment.
 
-While the shared Anaconda environment encapsulated in the module contains many of the most commonly used Python libraries for our users, you may still encounter a scenario in which you need to extend the functionality of the environment (i.e. install additional packages)
```

Comment:
just curious, why delete this?

Reply:
honestly I think it maybe just sounded a bit wordy ?? idk